src/index.js

```diff
@@ -1,15 +1,22 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
-import App from "./App";
-import { makeServer } from "../server";
+import App from "./frontend/App";
```

Comment:
Suggestion: App.js file should not be nested inside any folder. It should be located at the root level of your project.

Reply:
Thanks for pointing it out himadri